components/homme/src/theta-l/element_state.F90

```diff
@@ -69,6 +69,7 @@ module element_state
     real (kind=real_kind) :: FQ(np,np,nlev,qsize_d)                ! tracer forcing
     real (kind=real_kind) :: FM(np,np,3,nlev)                      ! momentum forcing
     real (kind=real_kind) :: FT(np,np,nlev)                        ! temperature forcing
+    real (kind=real_kind) :: FPHI(np,np,nlevp)                     ! PHI (NH) forcing
     real (kind=real_kind) :: FQps(np,np)                   ! forcing of FQ on ps_v
```

Comment:
minor point: FPHI at the bottom is always 0, so, for alignment/vectorization, would it be better to keep is of size nlev?